doc/Installation/Installation-Ubuntu-1604-Apache.md

```diff
@@ -17,8 +17,7 @@ path: blob/master/doc/
 #### Install LibreNMS
 
     cd /opt
-    composer create-project --no-dev --keep-vcs librenms/librenms librenms dev-master
-
+    export COMPOSER_ALLOW_SUPERUSER=1  && composer create-project --no-dev --keep-vcs librenms/librenms librenms dev-master
```

Comment:
This just disables the warning...  The warning is appropriate.
If we want to get rid of the warning, we could use git clone and composer install as separate steps.  Running the second as the librenms user.

Reply:
So it's oke to show a bigass warning, even if there is an option to disable it and make the installer look a little bit nicer?

Reply:
Silencing it is probably not the best course of action. @murrant idea would be better if you want to get rid of it. Further, docs state:

> It also disables automatic clearing of sudo sessions, so you should really only set this if you use Composer as super user at all times like in docker containers

which is clearly not the case here with LibreNMS

Reply:
Do i change the docs to git clone + composer install? or do i just set a message in the docs saying: "it's oke if you see a bigass warning"